tests/test_iot/test_server.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
+
+import sure  # noqa
+
+import moto.server as server
+from moto import mock_iot
+
+'''
+Test the different server responses
+'''
+
+@mock_iot
+def test_iot_list():
+    backend = server.create_backend_app("iot")
+    test_client = backend.test_client()
+    # do test
```

Comment:
@JackDanger said this to me many times :wink:, can we have a test here, all it would need to do is call a url, it just verifies the server comes up is all.

Something simple like this would be fine - https://github.com/spulec/moto/blob/master/tests/test_batch/test_server.py

Reply:
added test for checking if server is up.